utility/module.go

```diff
@@ -143,16 +141,32 @@ func (u *utilityModule) GetActorModules() map[string]modules.Module {
 	return u.actorModules
 }
 
-func (u *utilityModule) GetServicerModule() modules.ServicerModule {
-	return u.actorModules[servicer.ServicerModuleName]
+func (u *utilityModule) GetServicerModule() (modules.ServicerModule, error) {
+	m, ok := u.actorModules[servicer.ServicerModuleName].(modules.ServicerModule)
+	if !ok {
```

Comment:
OPTIONAL NIT:

I think doing the following is more readable and shorter, but not a blocker

```go
if m, ok := u.actorModules[servicer.ServicerModuleName].(modules.ServicerModule); ok {
	return m, nil
}
u.logger.Warn(...)
...
```